core/lib/server/pub-filters.db.test.ts

```diff
@@ -157,9 +157,9 @@ const seed = createSeed({
 			id: relationId,
 			pubType: "Basic Pub",
 			values: {
-				Relation: null,
 				Number: 50,
 			},
+			relatedPubs: {},
```

Comment:
this seed would fail after I made the changes to `normalizePubValues`. I think related pubs are meant to be declared via a separate `relatedPubs` key (outside of `values`) so I think this was originally not declared quite right (but also not used in the tests) but let me know if that's a bad assumption!